test-app-sitebricks.yaml

```diff
@@ -78,41 +78,43 @@ paths:
           description: "return balance of user successfully"
         400:
           description: "invalid user id"
-  /accounts/{userId}/deposit/{value}:
+  /accounts/{userId}/deposit:
     post:
       summary: "deposit amount to user account"
       operationId: "deposit"
       parameters:
+        - in: "body"
+          name: "amount"
+          description: "amount to deposit"
+          required: true
+          schema:
+            type: "integer"
```

Comment:
its like an value that user got online, for example that user play a game that game receive tokens, because he won the game or because he shared to someone that game and e.t.c. its not a money or something like that